net/tests/Sails.TestUtils/Containers/GearNodeContainer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using EnsureThat;
+
+namespace Sails.TestUtils.Containers;
+
+public sealed class GearNodeContainer : IAsyncDisposable
+{
+    public GearNodeContainer(string gearNodeVersion, bool reuse)
+    {
+        EnsureArg.IsNotNullOrWhiteSpace(gearNodeVersion, nameof(gearNodeVersion));
```

Comment:
i think, `version` should be optional parameter or builder method `WithVersion`
By default - use latest veriosn

Reply:
added TODO